fee/payload.go

```diff
@@ -80,5 +37,5 @@ func paymentDataSize(dataSize int64) int64 {
 }
 
 func PaymentDataFee(dataSize int64) *big.Int {
```

Comment:
This name is not good. It could be related to state data update/change
Something like `StateDataFee` or `StateChangeFee`

Reply:
Notice that the other function "paymentDataSize" is also used by payload, so not need to rename